tasks/SPARK/src/test/java/kmer/RetriableTest.java

```diff
@@ -0,0 +1,38 @@
+package kmer;
+
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class RetriableTest {
+
+    class TestRetriable implements Retriable<Integer> {
+        private int counter;
+
+        public TestRetriable() {
+            counter = 0;
+        }
+
+        public Integer run() throws Exception {
+            counter++;
+            if (counter < 2) {
+                throw new Exception("boom!");
+            }
+            return counter;
+        }
+
+        public int getCounter() {
+            return counter;
+        }
+    }
+
+    @Test
+    public void runWithRetriesShouldRetry() throws InterruptedException {
+        TestRetriable retriable = new TestRetriable();
+        Retriable.runWithRetries( 3, 0, retriable);
```

Comment:
Do you want to try a retry test that uses a mock web server to return a 503 (that's what the blue box is using to signal a try again later, right?)